src/app/tree-view/recursive-selectable-tree/recursive-selectable-tree.ts

```diff
@@ -92,14 +92,15 @@ export class RecursiveSelectableTreeDemo {
         name: "A1",
         selected: false,
         expanded: true,
-        children: [
```

Comment:
@jeeyun - Are forms like these something we can specify with a rule? Or are they `BasedOnStyle: Google`?